scripts/create-anvil-cred-file.py

```diff
@@ -0,0 +1,13 @@
+import os
```

Comment:
Can we reuse `../scripts/create-env-file.js` to create GCP cred files instead of creating another?